src/style.css

```
@@ -1,3 +1,74 @@
+html {
+  --color-text-hover: #00acdc;
+}
+
 body {
   margin: 0;
```

```suggestion
  margin: 0;
  font-family: "Roboto", sans-serif;
```
It's better to specify a font family here to get it inherited by other elements